.github/workflows/ci.yml

```diff
@@ -27,7 +27,7 @@ jobs:
             - name: ✅ Run tests
               run: composer test
 
-    phpcs:
+    cs:
```

Comment:
@bertramakers will have to change the required CI steps in the Github settings then

Reply:
Could that also be the reason why it keeps hanging?

Reply:
`Code style` is already/still enabled. I think this has to be merged first so the reference to `phpcs` gets removed on the master branch.

Reply:
Ah no, it looks like it uses the `name`. Awesome.

Reply:
Okay!